src/Workspaces/Core/Portable/Formatting/FormattingOptions.cs

```diff
@@ -1,23 +1,39 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
 using Microsoft.CodeAnalysis.Options;
 
 namespace Microsoft.CodeAnalysis.Formatting
 {
     public static class FormattingOptions
     {
-        public static PerLanguageOption<bool> UseTabs { get; } = new PerLanguageOption<bool>(nameof(FormattingOptions), nameof(UseTabs), defaultValue: false);
+        public static PerLanguageOption<bool> UseTabs { get; } = new PerLanguageOption<bool>(nameof(FormattingOptions), nameof(UseTabs), defaultValue: false,
+            storageLocations: new EditorConfigStorageLocation("indent_style", s => s == "tab"));
```

Comment:
what's the intuition on these values? are they basically waht we expect the actual .editorConfig file to contain?